pkg/agent/clusterclaim.go

```diff
@@ -107,3 +121,55 @@ func generateClusterClientFromSecret(secret *corev1.Secret) (clusterclientset.In
 
 	return clusterClient, nil
 }
+
+// As the number of hosted cluster count reaches the max and threshold hosted cluster counts
+// are used to generate two cluster claims:
+// "hostedclustercount.above.threshold.hypershift.openshift.io" = true when the count > threshold
+// "hostedclustercount.full.hypershift.openshift.io" = true when the count >= max
+// Both max and threshold numbers should be valid positive integer numbers and max >= threshold.
+// If not, they default to 80 max and 60 threshold.
+func (c *agentController) getMaxAndThresholdHCCount() (int, int) {
+	maxNum := 80
```

Comment:
Repeated use of 80 probably should be a constant some where.

Reply:
yes